apps/chat-e2e/src/assertions/folderAssertion.ts

```diff
@@ -418,4 +424,51 @@ export class FolderAssertion<T extends Folders> {
       .soft(actualFoldersCount, ExpectedMessages.foldersCountIsValid)
       .toBe(expectedCount);
   }
+
+  public async assertFoldersSorting(sorting: Sorting) {
+    const allFolderNames = await this.folder.getFolderNames();
+    const sortedNames = allFolderNames.slice().sort((a, b) => {
+      const nameA = a.toLowerCase();
+      const nameB = b.toLowerCase();
```

Comment:
This approach doesn't handle cases like ["Abc","abc"] or ["abC", "abc"] correctly.

Reply:
fixed